### PR TITLE
Update mako bundesinv isos

### PR DESCRIPTION
**Pull request by pfanguin**
Test link object Zurich: https://mf-geoadmin3.dev.bgdi.ch/ltfap/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Ffap_bak_bundesinv&lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bak.bundesinventar-schuetzenswerte-ortsbilder&X=247250.08&Y=683942.92&zoom=7

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/ltfap/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Ffap_bak_bundesinv&lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bak.bundesinventar-schuetzenswerte-ortsbilder


Change : 
- link for pdf  -> data.geo.admin.ch

**Comment by loicgasser** [comment]
@pfanguin I moved data geoadmin in settings.

**Comment by pfanguin** [comment]
jenkins test

File: Makefile [hunk 1]
```diff
@@ -335,6 +336,7 @@ production.ini: production.ini.in
 		--var "shortener_allowed_hosts=$(SHORTENER_ALLOWED_HOSTS)" \
 		--var "vector_bucket=$(VECTOR_BUCKET)" \
 		--var "vector_profilename=$(VECTOR_PROFILENAME)" \
+        --var "datageoadminhost=$(DATAGEOADMINHOST)" \
```
Comment (loicgasser) [inline Makefile]: Indent issue, one tab and then spaces only

Comment (loicgasser) [inline Makefile]: Indent issue, one tab and then spaces only

File: chsdi/templates/htmlpopup/hydrogeologische-karte_100.mako [hunk 2]
```diff
@@ -4,21 +4,22 @@
...
 %>
 <tr>
   <td class="cell-left">${_('tt_document')}</td>
   <td class="cell-meta-big">
-% if pdf_link != '':
-% for link in pdf_link:
-        <a href="http://data.geo.admin.ch/ch.bafu.hydrogeologische-karte_100/legends/${link}"  target="_blank">${link}</a><br />
+% if pdf_links != '':
+% for link in pdf_links:
+        <a href="https://${dataHost}/${dataPath}/${link}"  target="_blank">${link}</a><br />
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/hydrogeologische-karte_100.mako]: I think you can leave it agnostic.

File: chsdi/templates/htmlpopup/isos.mako [hunk 3]
```diff
@@ -38,6 +45,14 @@
             &nbsp;-
         % endif
       </td>
+      % elif c['attributes']['pdf_dokument_1'] == 'ISOS_5800' :
+      <td class="cell-left" style="vertical-align: top;">PDFs</td>
+      <td> 
+          % for name in pdfname:
+          <a href="https://${dataHost}/${dataPath}/ISOS_5800_${name}.pdf"  target="_blank">ISOS_5800_${name}.pdf<a/><br>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/isos.mako]: Agnostic again

File: chsdi/templates/htmlpopup/isos.mako [hunk 4]
```diff
@@ -17,14 +17,21 @@
...
+url_pdf = None
+url_pdf2 = None
+if c['attributes']['pdf_dokument_1']:
+  url_pdf = '//%s/%s/%s.pdf' % (dataHost, dataPath, c['attributes']['pdf_dokument_1'])
+if c['attributes']['pdf_dokument_2']:
+  url_pdf2 = '//%s/%s/%s.pdf' % (dataHost, dataPath, c['attributes']['pdf_dokument_2'])
 
+pdfname = []
+if c['attributes']['pdfspecial'] is not None:
+    pdfname = [l for l in c['attributes']['pdfspecial'].split(',')]
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/isos.mako]: pdfname = c['attributes']['pdfspecial'].split(',') or an array column in DB

**Comment by loicgasser** [comment]
rebased.
I tested all 3 layers and it works well:
ch.astra.ivs-nat
ch.bak.bundesinventar-schuetzenswerte-ortsbilder
ch.bafu.hydrogeologische-karte_100

**Pull request merged**